### PR TITLE
OI-222: Fixed from the message template. Reworked local tasks for the…

### PR DESCRIPTION
**Pull request by nvelychenko**
… user page. Enable personal contact form. Create OpenideaL Full Date format. Reworked the user full view mode, created new layout template for it. Created logout link block. Rename "My.." block in user page. Made the footer block fetch image from openideal.com site. Hid slideshow icons when slide only 1. Fixed alignment of default user forms. Fixed alert. Uninstalled content translation module.

### CODE REVIEW

File: themes/openideal_theme/openideal_theme.theme [hunk 1]
```diff
@@ -110,7 +117,9 @@ function openideal_theme_theme_suggestions_comment_alter(array &$suggestions, ar
 function openideal_theme_preprocess_field__created(&$variables) {
```
Comment (db-Matroskeen) [inline themes/openideal_theme/openideal_theme.theme]: Don't we need to restrict it only to user entity?

File: themes/openideal_theme/scss/components/_homepage-teaser.scss [hunk 2]
```diff
@@ -170,6 +174,7 @@
   // Follow
   .flag-follow a,
   .homepage-teaser--flag--element__text {
+    width: 87px;
```
Comment (db-Matroskeen) [inline themes/openideal_theme/scss/components/_homepage-teaser.scss]: Should we change to `min-width`?

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]

File: themes/openideal_theme/scss/components/_homepage-teaser.scss (hunk 2 above)
Comment (nvelychenko) [reply themes/openideal_theme/scss/components/_homepage-teaser.scss]: Done.

File: themes/openideal_theme/openideal_theme.theme (hunk 1 above)
Comment (nvelychenko) [reply themes/openideal_theme/openideal_theme.theme]: Yeah, restricted.

**Pull request merged**